transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpBaseSecurityFilter.java

```diff
@@ -122,8 +122,10 @@ protected void writeChallenge(final HttpResponseMessage httpResponse,
     protected Collection<String> getAuthorizedRoles(Subject subject) {
         Set<Principal> principals = subject.getPrincipals();
         Collection<String> authorizedRoles = new HashSet<>();
+        logger.trace("Setting authorizedRoles");
```

Comment:
Is this needed? and if so why.  If it is it should have the session, but I'm assuming this is not needed